doc/content/gateways/concepts/lora-basics-station/cups-forwarding.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "CUPS Forwarding"
+description: ""
+weight: -1
+---
+
+CUPS redirecting makes it possible to redirect your gateway from one gateway server to another in an easy manner, thus facilitating remote gateway management.
+
```

Comment:
We are not actually forwarding the CUPS (which means moving the gateway to a different CUPS).

This is "Using CUPS to connect to a different LNS"